### ISSUE TITLE
upsampling/deconvolution

### ISSUE THREAD

**Issue by nnop** [nnop/notes#213]
**upsampling/deconvolution**

- [Upsampling and Image Segmentation with Tensorflow and TF-Slim](http://warmspringwinds.github.io/tensorflow/tf-slim/2016/11/22/upsampling-and-image-segmentation-with-tensorflow-and-tf-slim/)
- Refactor convolution layer and add deconvolution layer #1615
- BilinearUpsamplingFiller used with DeconvolutionLayer for upsampling #2213
- [Why loss is remaining constant during training FCN-8s?](https://stackoverflow.com/questions/41611130/why-loss-is-remaining-constant-during-training-fcn-8s)
- [FCN网络中使用的caffe类型层汇总--Convolution/Deconvolution/Crop/Eltwise/SoftmaxWithLoss](https://blog.csdn.net/laingliang/article/details/55048937)
- [caffe 关于Deconvolution的初始化注意事项](https://blog.csdn.net/xiamentingtao/article/details/79397350)

**Comment by nnop** [comment]
**pixel shuffle**

- [Wenzhe Shi（施闻哲）](https://scholar.google.com/citations?user=cQYo4SkAAAAJ&hl=en)
- [Real-Time Single Image and Video Super-Resolution Using an Efficient Sub-Pixel Convolutional Neural Network](http://cn.arxiv.org/pdf/1609.05158.pdf)
- [subpixel conv == transposed conv](https://www.mitbbs.com/article_t/Programming/31502731.html)

**Comment by nnop** [comment]
**checkerboard effect**

- [Oct. 17 2016] Deconvolution and Checkerboard Artifacts
- [1707.02937] Checkerboard artifact free sub-pixel convolution
- [ICLR 2018 - Pixel Deconvolutional Networks](https://openreview.net/forum?id=B1spAqUp-)
